src/backend/src/guards.rs

```diff
@@ -11,7 +11,8 @@ pub fn caller_is_not_anonymous() -> Result<(), String> {
 }
 
 pub fn caller_is_allowed() -> Result<(), String> {
-    if read_config(|s| s.allowed_callers.contains(&caller())) {
+    let caller = caller();
+    if read_config(|s| s.allowed_callers.contains(&caller)) || ic_cdk::api::is_controller(&caller) {
```

Comment:
This is the main change in this PR.  Controllers are now allowed.

Reply:
NTH: For consistency, just `is_controller` - i.e. using the function at the top (don't know how you call that)

```
use ... etc;
use ic_cdk::caller;
use ic_cdk::api::is_controller;

pub fn caller_is_allowed() -> Result<(), String> {
   let caller = caller();
   
   if read_config(|s| s.allowed_callers.contains(&caller)) || is_controller(&caller) {
```

Reply:
Done